projects/wvr-elements/src/lib/wvr-button/wvr-button.component.scss

```diff
@@ -3,236 +3,303 @@
 :host {
   --wvr-btn-border-radius: 5px;
   --wvr-btn-outline-background: transparent;
+  --wvr-btn-cursor-pointer: pointer;
+  --wvr-btn-font-size: 1rem;
+  --wvr-btn-line-height: 1.5;
+  --wvr-btn-padding: 0.375rem .75rem;
+  --wvr-btn-text-align: center;
+  --wvr-btn-vertical-align: middle;
 
```

Comment:
Will need to add `--wvr-btn-font-family-sans-serif`

Reply:
Added property "--wvr-btn-font-family-sans-serif"